src/state/opamState.ml

```diff
@@ -1999,19 +1999,19 @@ let add_to_env t ?opam (env: env) ?(variables=OpamVariable.Map.empty) (updates:
   env @ expand_env t ?opam updates variables
 
 let env_updates ~opamswitch ?(force_path=false) t =
-  let comp = compiler_comp t t.compiler in
-
   let add_to_path = OpamPath.Switch.bin t.root t.switch t.switch_config in
   let new_path =
     "PATH",
     (if force_path then "+=" else "=+="),
     OpamFilename.Dir.to_string add_to_path in
+(* Todo: put these back into their packages !
   let perl5 = OpamPackage.Name.of_string "perl5" in
```

Comment:
See cd9c8ea2 for the explanation; now we can get rid of the hack in opam and have this put back in the specific package that needs it!
PATH and MANPATH should probably be the only variables defined from opam, besides of OPAM* variables (well, actually, even those could be defined by the compilers...)